pre-commit.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+from typing import List
+import subprocess
+import sys
+
+RED = "\033[0;31m"
+GREEN = "\033[0;33m"
+CYAN = "\033[0;36m"
+NC = "\033[0m"  # No Color
+
+
+def fix_inplace() -> bool:
+    if len(sys.argv) == 2 and sys.argv[1] == "--fix-inplace":
```

Comment:
nitpick: this is fine, but `len(sys.argv) >= 2` (or even argparse) would definitely be more durable.

Reply:
I've decided to do it that way just because using argparse for single flag argument looked too bulky, now there is a second argument (for color) so I decided to switch to argparse :)